app/forms/household_relationship_form.py

```diff
@@ -17,16 +18,17 @@ def build_relationship_choices(answer_store, group_instance):
     :param group_instance: The instance of the group being iterated over
     :return:
     """
-    household_answers = answer_store.filter(block_id='household-composition')
+    household_first_name_answers = answer_store.filter(block_id='household-composition', answer_id='first-name', limit=settings.EQ_MAX_NUM_REPEATS)
```

Comment:
Ah yes, but I think if I remember correctly we are storing blank answers for this particular form where there's nothing entered, so probably not a problem.